Client/components/MessageBoard.jsx

```diff
@@ -0,0 +1,157 @@
+import React, {Component} from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom';
+import io from 'socket.io-client';
+import '../public/style.css';
+import $ from '../public/jquery.js';
+import AppActions from '../actions/AppActions';
+import GroupList from './GroupList';
+import UserList from './UserList';
+import MessageList from './MessageList';
+import TitleBar from './TitleBar';
+import MessageForm from './MessageForm';
+import GroupForm from './GroupForm';
+import AppAPI from '../utils/appAPI';
+import AppStore from '../stores/AppStore';
+import {AppBar, FlatButton} from 'material-ui'
+import Notification from './notification';
+import Drawer from 'material-ui/Drawer';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
+import IconButton from 'material-ui/IconButton';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import {green100, green500, green700} from 'material-ui/styles/colors';
+
+const style = {margin: 5};
+const muiTheme = getMuiTheme({
+  palette: {
+    primary1Color: green500,
+    primary2Color: green700,
+    primary3Color: green100,
+  },
+}, {
+  avatar: {
+    borderColor: null,
+  },
+});
+
+/**
+ * Gets initial state of the app
+ * @returns {void} returns void
+ */
+function getAppState() {
+    return {
+      errors: AppStore.getErrors(),
+      success: AppStore.getSuccess(),
+      loggedInUser: AppStore.getLoggedInUser(),
+      registeredUser: AppStore.getRegisteredUser(),
+      users: AppStore.getUsersNotInGroup(),
+      groups: AppStore.getUserGroups(),
+      messages: AppStore.getGroupMessages(),
+      selectedGroup: AppStore.getSelectedGroup(),
+      notifiedGroup: '',
+      userReadMessages: AppStore.getUsersReadMessage(),
+      open: false
+    };
+}
+
+class MessageBoard extends Component {
+
+/**
+ * 
+ * returns initial state of the application
+ * @returns {void} returns void
+ * @memberof MessageBoard
+ */
+getInitialState(){
+      return getAppState();
+  }
+
+
+  connect(){
+  }
+
+  /**
+   * Sets open state to false
+   * @return {void} return void
+   * @memberof MessageBoard
+   */
+  handleRequestClose() {
```

Comment:
I'm not sure if you're supposed to use es6, if you are please change your functions to the arrow style function